setup.py

```diff
@@ -22,13 +25,21 @@
 with open("mitiq/_version.py", "w") as f:
     f.write(f"__version__ = '{__version__}'\n")
 
+third_party_integration_requirements = {
```

Comment:
To confirm, when a user runs `pip install -e .[development]` or `make install`, this will install all the third party integrations as well, correct?

Reply:
How are we enforcing the version compatibility of the various 3P requirements? Do pip or setuptools do this automatically?

Reply:
> this will install all the third party integrations as well, correct?
Yes, exactly the same as before.

> How are we enforcing the version compatibility of the various 3P requirements?
Same as we do now, that is, having all of them together in the development environment and running the test suite in such environment.